components/expenses/PayoutBankInformationForm.js

```diff
@@ -211,14 +211,20 @@ FieldGroup.propTypes = {
 };
 
 const DetailsForm = ({ disabled, getFieldName, formik, host, currency }) => {
+  const [accountDetails, setAccountDetails] = useState(null);
+  useEffect(() => {
+    const data = get({ ...formik.values }, getFieldName('data'));
+    setAccountDetails(data);
+  }, []);
```

Comment:
Why not simply:

```suggestion
  const [accountDetails, setAccountDetails] = useState(() => get(formik.values, getFieldName('data')));
```

Reply:
I think we don't need this `useState` at all. I've simplified the code. 😉